toolkit/resources/assets/isomaker/iso_root_arch-dependent_files/arm64/isolinux/isolinux.cfg

```diff
@@ -0,0 +1,7 @@
+# D-I config version 2.0
```

Comment:
In the summary for this PR please fill the last part describing how you validated the change.

Reply:
Updated steps on how I validated changes.